postinstall.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var process = require('process');
+var path = require('path');
+var VERSION = require('./node_modules/electron/package.json').version;
+var gypPath = path.join(__dirname, 'node_modules/.bin') + "/node-gyp";
+
+var exec = require('child_process').exec,
+  cmd = [
+    gypPath,
+    "rebuild",
+    "--target=" + VERSION,
+    "--arch=x64",
+    "--dist-url=https://atom.io/download/atom-shell",
+    "--msvs_version=2013"
```

Comment:
this can and should be automatically set inside of `.npmrc`. when you do something like `npm install -g windows-build-tools` within an elevated PowerShell (the recommended way to do this these days) it will do this automatically for you. I am uneasy about requiring only one version of msvs.